### PR TITLE
[netatmo] Enhance bridge status reporting

### PR DESCRIPTION
**Pull request by clinique**
This small PR just reports the granting path in the status of the bridge so the user does not have to search in the log to find the path to grant the binding against the API

image

Extracted from #16489

### CODE REVIEW

**Comment by clinique** [comment]
@jlaur : this one is ready for your review.

File: bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/i18n/netatmo.properties [hunk 1]
```diff
@@ -459,7 +459,7 @@ config.refreshInterval.description = The refresh interval to poll Netatmo API (i
 
 conf-error-no-client-id = Cannot connect to Netatmo bridge as no client id is available in the configuration
 conf-error-no-client-secret = Cannot connect to Netatmo bridge as no client secret is available in the configuration
-conf-error-grant-needed = Configuration incomplete, please grant the binding to Netatmo Connect.
+conf-error-grant-needed = Complete the configuration by granting the binding to Netatmo Connect : `{0}`.
```
Comment (jlaur) [inline bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/i18n/netatmo.properties:462]:
```suggestion
conf-error-grant-needed = Complete the configuration by granting the binding to Netatmo Connect: ''{0}''.
```

Three things:
- Removed space before colon.
- Replaced **`** by **'** which seems more usual, WDYT?
- At least when using **'**, the placeholder has to be escaped (see https://github.com/openhab/openhab-addons/issues/15460#issuecomment-1686928930)

**Review by jlaur — CHANGES_REQUESTED** [review]
Thanks! Only one minor comment, otherwise LGTM. @lolodomo - would you like to have a look as binding maintainer?

**Review by jlaur — APPROVED** [review]
LGTM. @lolodomo - would you like to have a look as well?

**Pull request merged**